vocabs/DID-core-shape.ttl

```diff
@@ -0,0 +1,129 @@
+# You can run this, for example, with:
+#  pyshacl -s DID-core-shape.ttl -e DID-core.ttl -f human test.ttl
+#
+
+@prefix did:  <https://w3id.org/did#> .
```

Comment:
are we sure w3id.org is the right choice for this prefix? 

it currently redirect without preserving fragment... https://w3id.org/did#dfn-alsoknownas

I would have thought this would point to vocabulary file, like:

```
@prefix as:   <https://www.w3.org/ns/activitystreams> .
```

does.

Reply:
This is what is in the `@context` file... (a.k.a. don't kill the messenger :-)

Reply:
killing messengers results in better messages ;)

@rhiaro @msporny I won't accept this PR as is :)

Reply:
@OR13 -- I would say "yes, w3id.org is the right choice". You set up the w3id.org redirects for `did-resolution`.  You should know that fixing this issue of the redirect-without-fragment (which might just be to make the redirect target retain the fragment, as in https://w3c.github.io/did-core/#dfn-alsoknownas) just requires an issue and/or PR against <https://github.com/perma-id/w3id.org/blob/master/did/.htaccess>. PR best to come from a past contributor to that (who might also add a README to describe the DID redirect and indicate who may make future changes to the `did` folder of the https://github.com/perma-id/w3id.org/ repo), i.e., @dlongley, @msporny, or @davidlehn.

Reply:
@iherman Which context file are you looking at? The [current v1 here](https://github.com/w3c/did-spec-registries/blob/master/contexts/did-v1.jsonld) does not use w3id.org at all

Reply:
Hm. You are right (maybe I looked at an older file, I am not sure). But there is a problem nevertheless. The `@context` file says:.

```
"didns": "https://www.w3.org/ns/did/v1#"
```

but if I dereference that URL I get to the `@context` file. That is incorrect. A JSON-LD `@context` file is _not_ a vocabulary file. What I expect to see at that URL is the RDFS definition of the terms, i.e., what I have put into this PR.

Reply:
@iherman yes, I think we need a separate conversation about putting the right files under the right places at /ns/ and sorting out the conneg. We also have a html namespace file now that needs to go there at some point.

Reply:
> Which context file are you looking at? The current v1 here does not use w3id.org at all

B.t.w., I just realized that the context file has been changed since I branched off... hence the discrepancy.